centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -44,6 +44,7 @@ Options getOpts() {
 
     private Options opts;
     private String token = "";
+    private com.google.protobuf.ByteString connectData;
```

Comment:
I suppose it's better to use `byte[]` like we do [in RPC method](https://github.com/centrifugal/centrifuge-java/blob/master/centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java#L786) for example?

Reply:
ah sorry, I am wrong - you already use it right.